packages/module/src/components/contextmenu/ContextSubMenuItem.tsx

```diff
@@ -11,6 +11,11 @@ interface ContextSubMenuItemProps {
   children: React.ReactNode[];
 }
 
+/** Check if an event target is also a node. Needed to stop runtime errors where the target does not implement Node. */
+const isNode = (node: any): boolean => {
```

Comment:
Can we rename this to `isHTMLNode` or something not to be confused with a Topology `Node`?

Reply:
Sure! I renamed it to `implementsDOMNode`